jadx.desktop

```
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Name=JADX
+Comment=Dex to Java decompiler
+Type=Application
+Categories=Application;Development;
+Exec=/app/bin/jadx-gui
```

```suggestion
Exec=jadx-gui
```

it's never an absolute path.